Library/Homebrew/cask/lib/hbc/artifact/pkg.rb

```diff
@@ -52,7 +54,24 @@ def run_installer(pkg_description)
         ]
         args << "-verboseR" if Hbc.verbose
         args << "-allowUntrusted" if pkg_install_opts :allow_untrusted
-        @command.run!("/usr/sbin/installer", sudo: true, args: args, print_stdout: true)
+        with_choices_file pkg_install_opts(:choices) do |choices_path|
+          args << "-applyChoiceChangesXML" << choices_path if choices_path
+          @command.run!("/usr/sbin/installer", sudo: true, args: args, print_stdout: true)
+        end
+      end
+
+      def with_choices_file(choices)
```

Comment:
You can remove the `choices` argument and directly use `pkg_install_opts(:choices)` since this is an instance method.